packages/core/src/accordion/accordion-content.element.ts

```diff
@@ -39,9 +39,16 @@ import styles from './accordion-content.element.scss';
  * @cssprop --background
  */
 export class CdsAccordionContent extends LitElement {
+  @state({ type: String, reflect: true, attribute: 'role' })
+  protected role = 'region';
+
   connectedCallback() {
     super.connectedCallback();
     this.setAttribute('slot', 'accordion-content');
+
+    if (!this.id) {
```

Comment:
Does id need to be a property?

Reply:
I think id is inherited from LitElement but now I can't find confirmation of that from the docs.

Reply:
`id` is an inherited property because `LitElement` extends `HTMLElement`. So all our components automatically get `id` without the need for `@property`

Reply:
I agree that `id` is part of the DOM spec for HTMLElement. My question was more concerned about whether we want this to be Lit property because of the tie-in to the update lifecycle and the React wrapper. React already has problems with us updating the DOM without it knowing. So I tend to only do this (updating the DOM outside of the Lit lifecycle) for stealth attributes that drive CSS selectors or internal logic (like with Clarity Motion). Things React wouldn't need to keep track of in the Virtual DOM.

Reply:
I also updated the tree-view PR to not be a Lit property per @coryrylan 's suggestion. Either way we pick, we should keep it consistent across components which way we go.

Reply:
Yeah all good points @mathisscott  So many DOM props/attr are not consistent in staying in sync together.